### ISSUE TITLE
[bug] Could not satisfy explicit device specification '/device:GPU:0' because no supported kernel for GPU devices is available

### ISSUE THREAD

**Issue by hyybuaa** [tensorflow/models#4548]
InvalidArgumentError (see above for traceback): Cannot assign a device for operation 'InceptionV4/Logits/Predictions': Could not satisfy explicit device specification '/device:GPU:0' because no supported kernel for GPU devices is available.
         [[Node: InceptionV4/Logits/Predictions = Softmax[T=DT_FLOAT, _device="/device:GPU:0"](InceptionV4/Logits/Logits/BiasAdd)]]

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by qlzh727** [comment]
What's the model you are testing with when you see this error? Please also provide the information requested above for troubleshooting.

**Comment by qlzh727** [comment]
Assigning to slim model owner.

**Comment by Anubhav2000** [comment]
@tensorflowbutler I am dealing with the same problem.Here are the specifications you asked for.
Custom Code-Yes
Windows 10 pro 64-bit
Tensorflow installed from command prompt using pip3
Tensorflow Version-1.8.0
Bazel Not Installed
Cuda 9.0,Cudnn 7.1.4
Nvidia 940mx 2 GB
with tf.Session() as sess:
    writer=tf.summary.FileWriter("output",sess.graph)
    print(sess.run(c))
    writer.close()

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be considerably old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
